services/management/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Service Management packages for Go
 
-The `github.com/Azure/azure-sdk-for-go/services/servicemanagement` packages are used to perform operations using the Azure Service Management (ASM), aka classic deployment model. Read more about [Azure Resource Manager vs. classic deployment](https://azure.microsoft.com/documentation/articles/resource-manager-deployment-model/). Packages for Azure Resource Manager are in the [arm](../arm) folder.
+The `github.com/Azure/azure-sdk-for-go/services/management` packages are used to perform operations using the Azure Service Management (ASM), aka classic deployment model. Read more about [Azure Resource Manager vs. classic deployment](https://azure.microsoft.com/documentation/articles/resource-manager-deployment-model/). Packages for Azure Resource Manager are in the [arm](../arm) folder.
```

Comment:
🤔 with a generic name like `management` I could see some folks (read: me 😛) getting confused with something else - as such, would it make sense to name this package `classic` (or `classic/management`)?

Reply:
Makes sense to me!